src/Parser.php

```diff
@@ -334,7 +334,15 @@ public function addImportedSymbol($type, $alias, $name = null, AbstractExpressio
 
     public function getImportedSymbol($type, $alias)
     {
-        return isset($this->importedSymbols[0][$type][$alias]) ? $this->importedSymbols[0][$type][$alias] : null;
+        if (null !== $this->peekBlockStack()) {
+            foreach ($this->importedSymbols as $functions) {
```

Comment:
This code is making things work "by chance". Adding it back until we find a better solution (which will only be for 2.x anyway).

Reply:
I'm adding more test to cover more edge cases, especially for the interaction between macros and blocks (we had none).